enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -731,16 +715,6 @@ will be limited to the `openshift-ingress` namespace to avoid introducing comple
 Gateway API upstream is still defining guidelines for DNS record management (see
 [kubernetes-sigs/gateway-api#2627](https://github.com/kubernetes-sigs/gateway-api/issues/2627)).
 
-###### Gateway Merging Across Namespaces
```

Comment:
Why delete this?  Isn't there anything we could keep?

Reply:
My thinking originally was that this section was referring to a situation that's gone now, because we block Istio's implementation-specific Gateway merging mechanism, as that precludes the preemption problem described.

However after thinking about it again, I realize for posterity this would actually probably be a good place to explain what we've done in terms of blocking Istio's implementation, as that choice was due to what you might call a "security policy". I've added this section back but updated it to be current. LMKWYT :thinking: